r3bdata/neulandData/R3BNeulandCalData.h

```diff
@@ -21,11 +21,12 @@ class R3BNeulandCalData : public TObject
 {
   public:
     R3BNeulandCalData() = default;
-    R3BNeulandCalData(Int_t barId, Int_t side, Double_t time, Int_t qdc);
+    R3BNeulandCalData(Int_t barId, Int_t side, Double_t time, Double_t triggertime, Int_t qdc);
 
     Int_t GetBarId() const { return fBarId; }
     Int_t GetSide() const { return fSide; }
     Double_t GetTime() const { return fTime; }
```

Comment:
Redefine GetTime method to get triggered time. Also add another getter for untriggered time:
```suggestion
    Double_t GetTime() const { return fTime - fTriggerTime; }
    Double_t GetTimeUntriggered() const { return fTime; }
```